dotenv/load.ts

```diff
@@ -1,5 +1,47 @@
-// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+import { DenoEnv, parse } from "./mod.ts";
```

Comment:
```suggestion
import { type DenoEnv, parse } from "./mod.ts";
```

Use [`type` modifier on type import names](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-5.html#type-modifiers-on-import-names)